xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-index/src/main/java/org/xwiki/extension/index/security/SecurityVulnerabilityDescriptor.java

```diff
@@ -184,6 +188,33 @@ public SecurityVulnerabilityDescriptor setReviews(String reviews)
         return this;
     }
 
+    /**
+     * Retrieves the set of aliases associated with the current object.
+     *
+     * @return a set of aliases, used for vulnerability comparison, two vulnerabilities sharing an alias or an id are
+     *     considered equals
+     * @since 15.9RC1
+     */
+    @Unstable
+    public Set<String> getAliases()
+    {
+        return this.aliases;
```

Comment:
It would simply a lot of code to return an empty set when this.alias is null (i.e. never return null, which by the way is not documented in the javadoc currently).